src/main/resources/assets/actuallyadditions/lang/en_US.lang

```diff
@@ -579,6 +581,10 @@ tooltip.actuallyadditions.battery.changeMode=Sneak-right-click to toggle.
 tooltip.actuallyadditions.previouslyDoubleFurnace=Previously "Double Furnace"
 tooltip.actuallyadditions.previouslyBag=Previously "Bag"
 tooltip.actuallyadditions.previouslyVoidBag=Previously "Void Bag"
+tooltip.actuallyadditions.item_filling_wand.selectedBlock=Selected Block: %s
+tooltip.actuallyadditions.item_filling_wand.selectedBlock.none=None
+tooltip.actuallyadditions.itemEnergy=%s/%s Crystal Flux
```

Comment:
The String "Crystal Flux" should be hardcoded and not renameable, in my opinion.